awx/main/access.py

```diff
@@ -2824,6 +2824,13 @@ def can_add(self, data):
         else:
             return (self.check_related('workflow_approval_template', UnifiedJobTemplate, role_field='admin_role'))
 
+    def can_start(self, obj, validate_license=False):
+        # Super users can start any job
+        if self.user.is_superuser:
+            return True
+
+        return self.user in obj.workflow_job_template.execute_role
```

Comment:
@appuk @kdelee @beeankha @AlanCoding what do you think about this?

"You can only copy a workflow with pause nodes if the user has execute role access to the workflow."?

Reply:
In order to copy a WFJT, you need admin_role to its organization. So testing WFJT.execute_role is not much different from returning `True` so that it doesn't error anymore. I would not object to either. A comment like `# for WFJT copy` might be nice